cli/password.go

```diff
@@ -50,14 +44,15 @@ func askForExistingRepositoryPassword(out io.Writer) (string, error) {
 	return p1, nil
 }
 
+func (c *App) setPasswordFromToken(pwd string) {
+	c.password = pwd
+}
+
 func (c *App) getPasswordFromFlags(ctx context.Context, isNew, allowPersistent bool) (string, error) {
 	switch {
-	case globalPasswordFromToken != "":
-		// password extracted from connection token
-		return globalPasswordFromToken, nil
-	case globalPassword != "":
+	case c.password != "":
 		// password provided via --password flag or KOPIA_PASSWORD environment variable
-		return strings.TrimSpace(globalPassword), nil
+		return strings.TrimSpace(c.password), nil
```

Comment:
nit: we should document that surrounding spaces are trimmed from the specified password.